TSQL_Parser/TSQL_Parser/Clauses/TSQLUsingClause.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace TSQL.Clauses
+{
+	public class TSQLUsingClause : TSQLClause

```

Comment:
I do agree that USING is a clause. Hopefully they don't add this word somewhere else in the language at some point.